simulation/traffic_simulator/src/entity/pedestrian_entity.cpp

```diff
@@ -109,6 +109,16 @@ void PedestrianEntity::setTargetSpeed(double target_speed, bool continuous)
   target_speed_planner_.setTargetSpeed(target_speed, continuous);
 }
 
+auto PedestrianEntity::getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel
```

Comment:
```
auto PedestrianEntity::getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel
```
↓
```
auto PedestrianEntity::getDriverModel() const -> traffic_simulator_msgs::msg::DriverModel
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/650/commits/d3a8c09835cca7d0b623b86317f54f9152eb257e